src/crafting.cpp

```diff
@@ -2148,7 +2172,7 @@ void player::complete_disassemble( item_location &target, const recipe &dis )
             item newit( comp.type, calendar::turn );
             // Counted-by-charge items that can be disassembled individually
             // have their component count multiplied by the number of charges.
-            if( dis_item.count_by_charges() && dis.has_flag( flag_UNCRAFT_SINGLE_CHARGE ) ) {
+            if( dis_item.count_by_charges() && ( dis.has_flag( flag_UNCRAFT_SINGLE_CHARGE ) ) ) {
```

Comment:
What's that for?

Reply:
Ahhhh, that wasn't supposed to be left in! :(